### PR TITLE
feat: Serialize multiple categories and resources to one line

### PR DESCRIPTION
**Pull request by NRG-Drink**
It's allowed to have `CATEGORIES` property multiple times in an event but it's not that pretty and some other programs will have trouble reading such events. So it's more a feature than a bug (#156).
Here is a quick fix to display multiple categories in one line.

### CODE REVIEW

File: Ical.Net.Tests/SymmetricSerializationTests.cs [hunk 1]
```diff
@@ -238,7 +238,7 @@ public static IEnumerable UriAttachment_TestCases()
         yield return new TestCaseData("\\\\uncPath\\to\\resource.txt", new Uri("\\\\uncPath\\to\\resource.txt")).SetName("UNC path URL");
     }
 
-    [Test, Ignore("TODO: Fix CATEGORIES multiple serializations")]
+    [Test]
```
Comment (axunonb) [inline Ical.Net.Tests/SymmetricSerializationTests.cs]: Maybe assure also that the list contains expected Categories?
```cs
Assert.That(serialized, Does.Contain("CATEGORIES:Foo,Bar,Baz"));
```

File: Ical.Net/Serialization/PropertySerializer.cs [hunk 2]
```diff
@@ -36,15 +36,35 @@ public PropertySerializer(SerializationContext ctx) : base(ctx) { }
         // the property and parameter values
         var sf = GetService<ISerializerFactory>();
 
+        // TODO: Exhaust this list with all properties which can be displayed in one line.
```
Comment (axunonb) [inline Ical.Net/Serialization/PropertySerializer.cs:39]: Remove the TODO and for now add RESOURCES, as another frequent and simple use case?

**Comment by axunonb** [comment]
Thanks, very good! See comments if your time allows.

File: Ical.Net.Tests/SymmetricSerializationTests.cs (hunk 1 above)
Comment (NRG-Drink) [reply Ical.Net.Tests/SymmetricSerializationTests.cs]: I had this first but was not sure if the sequence matters or not.
The test is now extended as suggested and with another test-case (including null).

**Review by axunonb — APPROVED** [review]
That was fast, great! Thanks for your support.
Would you like to join the ical.net party?

**Comment by NRG-Drink** [comment]
> That was fast, great! Thanks for your support. Would you like to join the ical.net party?

You were even faster. I wasn't done and there is an error in the test 😅
btw. is it possible that ATTENDEE is broken? I tried to enable oneline for attendees too but it only performs a ToString on it and not serializing it properly. I think I will leave it for the moment.

**Comment by NRG-Drink** [comment]
> That was fast, great! Thanks for your support. Would you like to join the ical.net party?

I can try to help out on smaller tasks if it helps 😊

**Comment by axunonb** [comment]
> btw. is it possible that ATTENDEE is broken? I tried to enable oneline for attendees

`Attendee` requires its own `AttendeeSerializer`, as it's more complex than e.g. CATEGORIES. That's why.

> You were even faster. 

Should have awaited your review request, right
